kbs/src/api_server.rs

```diff
@@ -115,6 +123,11 @@ impl ApiServer {
                             .route(web::get().to(api))
                             .route(web::post().to(api)),
                     )
+                    .service(
+                        web::resource("/metrics")
+                            .route(web::get().to(prometheus_metrics))
+                            .route(web::post().to(|| HttpResponse::MethodNotAllowed())),
```

Comment:
I don't think this is required, is it?

Reply:
Quite possibly, yes.  Some code I saw used it and other routes handle `web::post()` too so I thought I'd include it for completeness but I'm happy to remove it.